Sources/SwiftKuery/Table.swift

```diff
@@ -19,7 +19,7 @@
 /// Subclasses of the Table class are metadata describing a table in a relational database that you want to work with.
 open class Table: Buildable {
     var _name = ""
-    private var columns: [Column]
+    public private (set) var columns: [Column]
```

Comment:
When the schema information is not available, in the default case we use `dateTimeFormatter`.
Should be possible to move parameter formatting to core, but would probably make the code much more complicated for MySQL. Not sure if it would be preferable for the other plugins, but am open to switching to something like that if it is beneficial for them. 

`columns` in `Insert` is already [public](https://github.com/IBM-Swift/Swift-Kuery/blob/master/Sources/SwiftKuery/Insert.swift#L25), so it seemed ok to me to make this publicly gettable and it was the shortest path for me.